libs/hermes/hermes.quiver/tests/unit/exporters/test_torch_onnx_exporter.py

```diff
@@ -43,3 +45,26 @@ def test_torch_onnx_exporter(temp_local_repo, torch_model):
     exporter._check_exposed_tensors("input", input_shapes)
     assert len(model2.config.input) == 1
     assert model2.config.input[0].name == "input_0"
+
+
+def test_torch_onnx_exporter_with_kwargs(temp_local_repo, torch_model):
```

Comment:
You'll have to mark this test with the `@pytest.mark.torch` decorator so that the CI only runs it when the torch extra is included in the install. Quiver tries to work with multiple different frameworks, and so rather than requiring all of them it has groups of extra installs (torch, tensorflow, tensorrt), different permutations of which get tested during CI, so tests need to be marked so that we only test the frameworks we expect an install to have.